packages/main/src/themes/TimePicker.css

```diff
@@ -9,6 +9,7 @@
 }
 
 :host {
+	min-width: calc(var(--_ui5_input_min_width) + var(--_ui5_input_icon_width));
```

Comment:
add TimePicker responsible team to review that

Reply:
did you?

Reply:
yes